zc_common/jwt_auth/permissions.py

```diff
@@ -0,0 +1,65 @@
+from rest_framework import permissions
+
+USER_ACTOR = 'user'
+STAFF_ACTOR = 'staff'
+SERVICE_ACTOR = 'service'
+ANONYMOUS_ACTOR = 'anonymous'
+
+USER_ROLES = [USER_ACTOR]
+STAFF_ROLES = [USER_ACTOR, STAFF_ACTOR]
+SERVICE_ROLES = [SERVICE_ACTOR]
+ANONYMOUS_ROLES = [ANONYMOUS_ACTOR]
+
+
+def is_staff(request):
+    return USER_ACTOR in request.user.roles and STAFF_ACTOR in request.user.roles
```

Comment:
A staff user can't not have the user role also. It's unnecessary to check for it.